chef/cookbooks/crowbar/templates/default/apache.conf.erb

```diff
@@ -9,7 +9,7 @@
     ErrorDocument 422 /422.html
     ErrorDocument 500 /500.html
 
-    ProxyPass / http://127.0.0.1:<%= @port %>/ timeout=600 Keepalive=On
+    ProxyPass / http://127.0.0.1:<%= @port %>/ timeout=<%= @proxy_timeout %> Keepalive=On
     ProxyPassReverse / http://127.0.0.1:<%= @port %>/
     SetEnv proxy-nokeepalive 1
```

Comment:
We have still `Keepalive=On` and `SetEnv proxy-nokeepalive 1` which makes no sense.